src/options.html

```diff
@@ -7,34 +7,37 @@
     <title>Settings page</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <link rel="stylesheet" href="css/options.css">
+    <link rel="stylesheet" type="text/css" href="css/ldcp.css"/>
+    <script type="text/javascript" src="js/ldcp.min.js"></script>
```

Comment:
What does the `ldcp` file do? What are we using it for? Let's try to reduce dependencies as much as possible. :smile:

Reply:
@zamhaq its for color picker, ldcp is color picker. Link to dependency: http://zbryikt.github.io/ldcolorpicker/